doc/how_to/interactivity/bind_function.md

```diff
@@ -0,0 +1,43 @@
+# Make your functions interactive
+
+After completing this, you can make your functions interactive by binding them to widgets. This is done with the use of `pn.bind`, which binds a function or method to the value of a widget.
+
+---
+
+## Bind your functions to widgets
```

Comment:
```suggestion
## Bind a function to a widget
```

Reply:
I think this should be framed the other way around, you bind a widget to a function.